.travis.yml

```diff
@@ -142,7 +142,7 @@ matrix:
     - os: osx
```

Comment:
Note, right now the default `osx` config is akin to `osx_image: xcode7.3` which uses macOS 10.11 (El Capitan).  Perhaps I should specify that, or add a builder on Sierra just to double-check things if we see problems.